graders/python/python_autograder/code_feedback.py

```diff
@@ -470,12 +498,27 @@ def bad(msg: str) -> Literal[False]:
         return True
 
     @classmethod
-    def call_user(cls, f: Callable[..., T], *args: Any, **kwargs: Any) -> T:
+    def call_user(
+        cls,
+        f: Callable[..., T],
+        *args: Any,
+        stop_on_exception: bool = True,
```

Comment:
Note that this is technically a breaking change, but only if an instructor is already passing in a kwarg with this argument name already. We could prefix this argument with an underscore (i.e. `_stop_on_exception`) to make this less likely, but that feels like overkill.